### ISSUE TITLE
the hub - search bar - missing form label

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#95]
WAVE is alerting to the search bar on https://rrchnm.org/the-hub/ missing a form label.

**Comment by kimisgold** [comment]
This is the same situation as #96, where the markup is handled by a third party. I've notified them of the lack of form label in the same accessibility thread.
